lib/msal-core/src/XHRClient.ts

```diff
@@ -30,8 +33,11 @@ export class XhrClient {
                 } catch (e) {
                     reject(this.handleError(xhr.responseText));
                 }
-
-                resolve(jsonResponse);
+                const response: XhrResponse = {
+                    statusCode: xhr.status,
+                    responseBody: jsonResponse
```

Comment:
I think this can just be `body`, so that it is `response.body` instead of `response.responseBody`.

Reply:
Updated